server/storage/memory.go

```diff
@@ -84,7 +84,9 @@ func (st *MemStorage) GetCurrent(gun, role string) (data []byte, err error) {
 func (st *MemStorage) GetChecksum(gun, role, checksum string) (data []byte, err error) {
 	st.lock.Lock()
 	defer st.lock.Unlock()
-	data, ok := st.checksums[gun][checksum]
+	id := entryKey(gun, role)
+	_, ok := st.tufMeta[id]
```

Comment:
Should there be an `ok` check here, or should it be assigned to a different variable?  It looks like the value would just get overwritten by the next line.